src/Umbraco.Cms.Integrations.OAuthProxy/Controllers/ShopifyComplianceController.cs

```diff
@@ -1,8 +1,9 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Integrations.OAuthProxy.Filters;
 
 namespace Umbraco.Cms.Integrations.OAuthProxy.Controllers
 {
-    [ApiController]
+	[ApiController]
```

Comment:
Your indenting has gone a bit off - here, and in the new class you have added.  You should align to use 4 spaces I think as per the rest of the repo.